job_service/repository/job_db.py

```diff
@@ -16,25 +17,39 @@
 
 
 class JobDb:
-    db = None
+    completed = None
+    in_progress = None
 
     def __init__(self):
         client = MongoClient(
             MONGODB_URL,
             username=MONGODB_USER,
-            password=MONGODB_PASSWORD
+            password=MONGODB_PASSWORD,
+            authSource="admin"
+        )
+        db = client.jobdb
+        self.completed = db.completed
+        self.in_progress = db.inprogress
+        # Creates an unique index on datasetName for inprogress collection
+        # if index not already present.
+        self.in_progress.create_index(
```

Comment:
👍

Reply:
This sadly caused issues with unit tests and the dependency tree :( I updated the database setup docs to include creating an index on the inprogress collection. The index info is stored in the persistent directory we will do backup on, so it should cause no issues to do it this way 👍🏻